src/mesh/meshblock.cpp

```diff
@@ -277,6 +275,7 @@ MeshBlock::MeshBlock(int igid, int ilid, Mesh *pm, ParameterInput *pin,
 
   return;
 }
+#endif
```

Comment:
If this whole block of code is commented out, can we delete it?

Reply:
Eventually, we need it back in in order to allow for restarting simulations.